sources/packages/api/src/route-controllers/institution/institution.aest.controller.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Body,
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Patch,
+  HttpStatus,
+} from "@nestjs/common";
+import { InstitutionService } from "../../services";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import {
+  InstitutionProfileDto,
+  InstitutionReadOnlyDto,
+} from "./models/institution.dto";
+import BaseController from "../BaseController";
+import { ClientTypeBaseRoute } from "../../types";
+import { InstitutionControllerService } from "./institution.controller.service";
+import { ApiTags, ApiOperation, ApiResponse } from "@nestjs/swagger";
```

Comment:
Can we add a blank line after the last import? It seems that almost all are having it.